src/components/ec-table-filter/ec-table-filter.story.js

```diff
@@ -211,6 +211,33 @@ const AllTemplate = storyArgs => ({
       component: EcTextFilter,
       isFullWidth: true,
     }];
+    const searchFilterFillingRemainingSpace = reactive([{
+      label: 'Payment status',
+      name: 'paymentStatus',
+      component: markRaw(EcSyncMultipleValuesFilter),
+      items: [{ text: 'Paid', value: 'paid' }, { text: 'Cancelled', value: 'canceled' }, { text: 'Overdue', value: 'overdue' }],
+    }, {
+      label: 'Supplier',
+      name: 'supplier',
+      component: markRaw(EcSyncMultipleValuesFilter),
+      items: [{ text: 'Supplier 1', value: 'supplier1' }, { text: 'Supplier 2', value: 'supplier2' }],
+      isSearchable: true,
+      isSelectAll: true,
+    }, {
+      label: 'Due date',
+      name: 'dueDate',
+      component: markRaw(EcDateRangeFilter),
+    }, {
+      label: 'Price',
+      name: 'price',
+      component: markRaw(EcCurrencyFilter),
+      comparisonSymbolItems,
+      currencyItems,
+    }, {
+      name: 'text',
+      class: 'lg:tw-col-auto tw-col-start-1 tw-col-end-[-2]',
```

Comment:
do you think we still need this class for filter? You can describe the grid in the layout class.

Reply:
in this case it's needed because we want that filter (search bar) to span through almost all columns when width is below `lg` threshold and, if I'm not wrong (because maybe I miss something), we need to apply those classes to the filter itself

Reply:
Filter could have full width of what container provides. Container for each filter is a grid cell, which we define using grid layout templates.

Reply:
No, I'd doubt we can use area names with Tailwind. I mean something like this.

![image](https://github.com/Ebury/chameleon/assets/5939240/39da13da-e16a-4fb5-95ab-889593a323df)

Reply:
The issue and the reason behind using those classes is that when using more than one row we can't set a different cell width, otherwise it will happen something like this:

![image](https://github.com/Ebury/chameleon/assets/77344353/66719bb3-fc8d-4316-9539-b41d8875dbab)

So that's why I use those classes to span the filter across the other columns:

![image](https://github.com/Ebury/chameleon/assets/77344353/95ac0f15-bcd9-4f84-ab1a-1112be1d9846)

I also thought about using subgrids but it was not possible in our case due to how we render the filters (and also because I don't knnow if we can use it with tailwind...)

Reply:
I see, okay, let's keep filter classes then.